backend/apis/flag-utils/is-first-degree-felony-flag-util.js

```diff
@@ -0,0 +1,19 @@
+exports.isFirstDegreeFelony = function isFirstDegreeFelony(parsedObj) {
+  if (
+    parsedObj.charges
+      .map(charge => charge.severity.toLowerCase())
+      .includes("1st degree felony")
+  ) {
+    if (
+      parsedObj.charges
+        .map(charge => charge.severity.toLowerCase())
+        .includes("amended")
+    ) {
+      return false;
+    } else {
+      return true;
+    }
+  } else {
+    return false;
+  }
```

Comment:
💡 I might have a suggestion for how we can make this a little more concise. overall though the idea looks good to me; stand by for a suggestion

Reply:
I'm happy to refactor if there is a more concise alternative. I'm new to Node, so I'm learning as I go. All suggestions will help me improve. Do you have a recommendation?

Reply:
What about something like this: 
```
const felonies = parsedObj.charges.filter(charge => (charge.severity.toLowerCase().includes("1st degree felony") && !charge.severity.toLowerCase().includes("amended"))
```
This will give you an array of all the charges that have "1 st degree felony" but that aren't amended. Then for the final return of the function you can do something like this: 
```
return !!felonies.length
```
If the felonies array has any items, it will return true, otherwise, 0 is falsey and it will return false. 

It doesn't have to be exactly that but hopefully that gives you something to go off, and I think it eliminates the bug where you're looping through the charges independently so the "amended" might not apply to the 1st degree felony. 

The whole func: 
```
exports.isFirstDegreeFelony = function isFirstDegreeFelony(parsedObj) {
  const felonies = parsedObj.charges.filter(charge => (charge.severity.toLowerCase().includes("1st degree felony") && !charge.severity.toLowerCase().includes("amended"))
  return !!felonies.length
}
```

Reply:
Yes, this makes more sense. If I loop independently, then a lesser severity charge (for example Misdemeanor) could be "amended" and the code would consider the "amended" and return false. I thought this may have been avoided with the nested if statements I was using, but it was still cumbersome code.